lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -426,6 +445,10 @@ def json_error(raw_response)
       def non_fractional_currency?(currency)
         CURRENCIES_WITHOUT_FRACTIONS.include?(currency.to_s)
       end
+
+      def emv_payment?(payment)
```

Comment:
So you'd prefer we go from: 
```ruby
class StripeGateway
...
def emv_payment?(payment)
  payment.respond_to?(:icc_data) && payment.icc_data.present?
end
```
to
```ruby
class StripeGateway
...
def emv_payment?(payment)
  payment.respond_to(:emv?) && payment.emv?
end
```
where
```ruby
class CreditCard
...
def emv? # also needs tests
  icc_data.present? 
end
```
? 
Doesn't seem better, but I might just have cabin fever. :disappointed_relieved: :